accounts/config.go

```diff
@@ -2,34 +2,43 @@ package accounts
 
 import (
 	"errors"
-	"log"
+	"fmt"
 	"path/filepath"
 
 	"github.com/phomer/scheduler/datastore"
+	"github.com/phomer/scheduler/log"
 )
 
 type ClientConfig struct {
+	Protocol    string
 	Hostname    string
 	Port        string
 	Username    string
 	Credentials string
+	Token       *Token
 }
 
 var paths = []string{".", "~/.schedule"}
 
 // Register initialization
-func NewClientConfig(username string, hostname string) *ClientConfig {
+func NewClientConfig(hostname string, username string, token *Token) *ClientConfig {
 
 	return &ClientConfig{
+		Protocol: "http",
 		Hostname: hostname,
 		Username: username,
 		Port:     "8000",
+		Token:    token,
 	}
 }
 
 // Client initialization
 func FindClientConfig() *ClientConfig {
-	file := FindFile(paths)
+	file, err := FindFile(paths)
```

Comment:
All this code here seems to be dealing with parsing client configuration file which adds a lot of complexity unnecessary to the challenge. Can the client just accept stuff it needs on the CLI?